schemas/3DScenesConfiguration/v1.0.0/3DScenesConfiguration.schema.json

```diff
@@ -271,6 +271,9 @@
                 "expression": {
                     "type": "string"
                 },
+                "propertyType": {
```

Comment:
I think with this change `propertyType` should be listed in the required field in L#281 below and we should generate the types from config again. Because it seems we set the propertyType always in the `TimeSeriesFormCallout` with the `expression.property.propertyType`.

Reply:
done

Reply:
@iremgokce This is only used in your history feature right? So it won't break existing configs that already exist. If we have any configs out there that don't have this value then it will fail to validate and crash on mount of the environment so we just have to be aware of that if we want to make it required.

Reply:
this is only the data history widget part

Reply:
so just a note though. Since _now_ the propertyType is a required field, it will break the config files with existing data history widget in it, fyi